### ISSUE TITLE
Installation parts do not have material description list

### ISSUE THREAD

**Issue by haubourg** [qwat/qwat-data-model#168]
Data coming from other models have such material lists. 
SIA 405 references : 
for connection parts between pipes
   - "inconnu"
   - "acier"
   - "fonte"
   - "matiere_synthetique"

for tanks:
  - "inconnu"
  - "beton"
  - "fibrociment"
   - "amiante_ciment"
   - "metal"
   - "pierre"
   - "matiere_synthetique"

and some other values for special constructions

**Comment by haubourg** [comment]
@varrieta, feel free to add any other values you find here.

**Comment by ponceta** [comment]
In Pully we have https://gist.github.com/ponceta/e79fcd36ca611e559fd20fdcd9d84769.

**Comment by haubourg** [comment]
@ponceta yes, I had that in mind for pipes, but here we face a question about materials for ponctual objects likes installation parts (and probably more like tanks). Hydrants already have material in QWAT.
